ufm-stream-to-fluentd/ufm-stream-to-fluentd.py

```diff
@@ -190,48 +181,12 @@ def check_app_params():
     global ufm_protocol
     global ufm_username
     global ufm_password
-
-    if args.fluentd_host:
-        fluentd_host = args.fluentd_host
-    elif get_config_value('fluentd-config', 'host'):
-        fluentd_host = CONFIG['fluentd-config']['host']
-    else:
-        raise ValueError("fluentd_host must be provided")
-
-    if args.fluentd_port:
-        fluentd_port = int(args.fluentd_port)
-    elif get_config_value('fluentd-config', 'port'):
-        fluentd_port = int(CONFIG['fluentd-config']['port'])
-    else:
-        raise ValueError("fluentd_port must be provided")
-
-    if args.ufm_host:
-        ufm_host = args.ufm_host
-    elif get_config_value('ufm-server-config', 'host'):
-        ufm_host = CONFIG['ufm-server-config']['host']
-    else:
-        raise ValueError("ufm_host must be provided")
-
-    if args.ufm_protocol:
-        ufm_protocol = args.ufm_protocol
-    elif get_config_value('ufm-server-config', 'ws_protocol'):
-        ufm_protocol = CONFIG['ufm-server-config']['ws_protocol']
-    else:
-        ufm_protocol = 'https'
-
-    if args.ufm_username:
-        ufm_username = args.ufm_username
-    elif get_config_value('ufm-server-config', 'username'):
-        ufm_username = CONFIG['ufm-server-config']['username']
-    else:
-        raise ValueError("ufm_username must be provided")
-
-    if args.ufm_password:
-        ufm_password = args.ufm_password
-    elif get_config_value('ufm-server-config', 'password'):
-        ufm_password = CONFIG['ufm-server-config']['password']
-    else:
-        raise ValueError("ufm_password must be provided")
+    fluentd_host = get_config_value(args.fluentd_host, 'fluentd-config', 'host')
```

Comment:
very cool
last arg should be good default or Null where we cannot provide good default